app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -199,6 +201,12 @@ def process_invalid_webauthn(form)
           link_html: t('errors.webauthn_setup.additional_methods_link'),
         )
       end
+
+      analytics.webauthn_setup_submitted(
+        platform_authenticator: form.platform_authenticator?,
+        errors: flash[:error],
```

Comment:
The method in `analytics_events.rb` is documented as receiving a hash, which is pretty conventional for `errors`, but `flash[:error]` would be a string here. For our existing Face/Touch Dashboard, I'm also unclear if this would have an impact on whether we're sending equivalent information to be able to represent the errors during setup.

Also, in some of the branches above we're assigning to `flash.now[:error]`, not `flash[:error]`, so I'm not sure if those would be passed through.

I'd wonder if we should either pass along an actual hash (from `form.to_h`? will this include equivalent detail to what we have in the dashboard already?), and/or a separate property for the single error computed for the flash.

Aside: I think this is an area where we probably want to refactor to reconsolidate error handling, since trying to manage it in multiple places between the `form.errors` and in here with `flash[:error]` creates a lot of confusion like this.

Reply:
I think either/or would be the best approach. Passing through the error hash is one thing we should do. If we wanted to do both or a separate value, then we do the logging through the setup form rather than the controller which should be fine.

Reply:
But now that I think about this some more, if we are going to refactor the error handling, let's pass the hash now, and then with the refactor, we can create the separate property? @aduth does this make sense?

Reply:
Yeah, I think so. As long as we have access to the same errors we're currently querying for in the CloudWatch dashboard.